### ISSUE TITLE
Suggestion: add warning when using BertForSequenceClassification without special [CLS] token

### ISSUE THREAD

**Issue by mboyanov** [huggingface/pytorch-pretrained-BERT#458]
Thank you for the awesome package!

As I understand it right now, it is the user's responsibility to add the special `CLS` and `SEP` tokens. People who haven't read the paper might miss this detail. 
It would be nice to issue a warning in the tokenizer or the model itself if the input is missing these tokens. An alternative is to include it in a more prominent location in the docs.
I understand that the package needs to be flexible so multiple architectures and tokenization procedures can be implemented, so I'm unsure where would be the best place for such a notification.

In short, my suggestion:
# GIVEN
A text sequence without the `CLS` tag
# WHEN
`BertForSequenceClassification` is called
# THEN
A warning is logged

**Comment by thomwolf** [comment]
I understand the issue but I'm not sure this would be very easy to implement as we would like to keep the model and tokenizer separated one from the other.

Do you have a solution in mind?

Otherwise, I'll guess people will have to continue to read the paper before using the model... 😉

**Comment by mboyanov** [comment]
I totally understand the need for separation between tokenizer and model. 
But, the specific `BertForSequenceClassification` does have the assumption and expects the input to be in a specific form. Even the docs mention "look at the preprocessing logic". 
I'd suggest extending the documentation for the model - sth like https://github.com/huggingface/pytorch-pretrained-BERT/pull/480
